client/app/components/CreateSourceDialog.jsx

```diff
@@ -45,6 +46,8 @@ class CreateSourceDialog extends React.Component {
     currentStep: StepEnum.SELECT_TYPE,
   };
 
+  formId = useUniqueId("sourceForm");
```

Comment:
Just wondering: how does it work with a class component? Hooks shouldn't work with class components, only with functional ones

Reply:
I didn't even think about it Lol. Will workaround it soon.